x-pack/legacy/plugins/task_manager/task_manager.test.ts

```diff
@@ -214,5 +214,30 @@ describe('TaskManager', () => {
 
       expect(claim).not.toHaveBeenCalled();
     });
+
+    test('handles failure due to inline scripts being disabled', () => {
```

Comment:
It would be useful to have a short comment here on how to set ES to cause this failure deliberately - both for trying it for real, and in case something in the response changes in the future that we need to change the test for

Reply:
Fair, I'll add that in.